docs/user-guides/workspace-access/zed.md

```diff
@@ -10,22 +10,66 @@ Use the Coder CLI to log in and configure SSH, then connect to your workspace wi
 1. [Install Zed](https://zed.dev/docs/)
 1. Install Coder CLI:
 
-   ```shell
+   <!-- copied from docs/install/cli.md make changes there -->
+
+   <div class="tabs">
+
+   ## Linux/macOS
+
+   Our install script is the fastest way to install Coder on Linux/macOS:
+
+   ```sh
    curl -L https://coder.com/install.sh | sh
    ```
 
-1. Log in and configure Coder SSH:
+   Refer to [GitHub releases](https://github.com/coder/coder/releases) for
+   alternate installation methods (e.g. standalone binaries, system packages).
+
+   ## Windows
+
+   > **Important:** If you plan to use the built-in PostgreSQL database, you will
```

Comment:
We should remove this section. It's only needed for running `coder server`. Sorry for not noticing it early. 

I can drop a Follow up.